app/Models/Bot.php

```diff
@@ -6,5 +6,7 @@
 
 class Bot extends Model
 {
-    //
+    public function webhooks() {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Models/Bot.php</br>**Line**: 9</br>Opening brace should be on a new line